packages/cli/internal/pkg/cli/context/info.go

```diff
@@ -2,10 +2,14 @@ package context
 
 import (
 	"reflect"
+	"strings"
 
 	"github.com/aws/amazon-genomics-cli/internal/pkg/cli/spec"
 )
 
+var serverEngines = map[string]bool{"cromwell": true}
+var headEngines = map[string]bool{"nextflow": true, "miniwdl": true, "snakemake": true}
```

Comment:
I would prefer to not have engine names as strings. We may want to have them as constants in some shared package. Here is an example of desired usage:

```go
var serverBasedEngine = map[engine.Name]bool{engine.NEXTFLOW: true}
```